sauron.el

```diff
@@ -42,7 +42,7 @@
      sauron-twittering sauron-jabber sauron-identica sauron-elfeed)
   "List of sauron modules to use. Currently supported are:
 sauron-erc, sauron-org and sauron-dbus, sauron-twittering,
-sauron-jabber, sauron-identica, sauron-elfeed.")
+sauron-jabber, sauron-identica, sauron-elfeed, sauron-ams-org.")
 
```

Comment:
This parts needs some slight massaging (to work with current git master).

Also, might want to add it to the Makefile.